pkg/strategy/pivotshort/trendema.go

```diff
@@ -21,6 +21,10 @@ type TrendEMA struct {
 func (s *TrendEMA) Bind(session *bbgo.ExchangeSession, orderExecutor *bbgo.GeneralOrderExecutor) {
 	symbol := orderExecutor.Position().Symbol
 	s.trendEWMA = session.StandardIndicatorSet(symbol).EWMA(s.IntervalWindow)
+	session.MarketDataStream.OnStart(func() {
+		s.trendEWMALast = s.trendEWMA.Values[s.trendEWMA.Length()-2]
```

Comment:
check the length first? preventing length return 0

Reply:
done